scss/functions/_get-color.scss

```diff
@@ -7,4 +7,13 @@
   // if they are valid, return the color
   // if they are not valid warn 'Invalid color type or level' and return null
   // null is a special value in Sass that means nothing, and it is used to represent the absence of a value, you need to return null if the type or level is not valid
+  @if map.has-key($colors, $type) {
+    @if map.has-key(map.get($colors, $type), $level) {
```

Comment:
saran saya kamu bisa menggabungkan nested `@if` di sini menjai 1 baris if dengan tambagan logic `and` itu akan membuat kode kamu lebih efisien

Reply:
ok coach updated